src/encoded/static/components/publication.js

```diff
@@ -26,7 +73,11 @@ const PublicationComponent = (props, reactContext) => {
     ];
 
     const crumbsReleased = (context.status === 'released');
-
+    const accessions = !context.datasets ? '' : context.datasets.reduce((dataset1, dataset2) => `${dataset1}&accession=${dataset2.accession}`, '');
+    const fields = ['accession', 'target', '@type', 'target', 'biosample_summary', 'lab', 'assay_term_name', 'assay', 'description', 'status'].reduce((field1, field2) => `${field1}&field=${field2}`, '');
```

Comment:
Could this be done in the module name space? The result wouldn’t ever change until the code is changed, so putting it in the module name space would let this happen only at page reload instead of every time a publication page is rendered.

Reply:
Done